diff --git a/v4.1.0-4ab2640ee7/hex/CMakeLists.txt b/v4.1.0-4ab2640ee7/hex/CMakeLists.txt
index ca08a4f..2c3e9e5 100644
--- a/v4.1.0-4ab2640ee7/hex/CMakeLists.txt
+++ b/v4.1.0-4ab2640ee7/hex/CMakeLists.txt
@@ -15,7 +15,8 @@ include(./cmake/util.cmake)
 find_program(MAKE "make")
 find_program(MERGEHEX "mergehex" HINTS "c:/Program Files (x86)/Nordic Semiconductor/nrf5x/bin")
 find_program(NRFUTIL "nrfutil")
-find_program(GIT "git")
+find_program(GIT "git" "git.cmd")
+
 
 if(NOT DEFINED CONNECTIVITY_VERSION)
     message(FATAL_ERROR "CONNECTIVITY_VERSION not provided, terminating compilation.")
